runtime/include/chpl-tasks-prvdata.h

```diff
@@ -23,7 +23,14 @@
 // This header file provides chpl_comm_taskPrvData_t
 #include "chpl-comm-task-decls.h"
 
-// The type for task private data
+// Type for Chapel-managed task private data
+// to be copied to new tasks.
+// Includes 'serial_state'
+typedef struct {
+  unsigned char data[32];
```

Comment:
If this is too small, the failures may be difficult to diagnose (segfaults, silent wrong answers, etc.).  Is there a guarantee or check anywhere that it is big enough?  The dependence between this and far-away Chapel module code makes me nervous.  Ideally we'd export this runtime type into the module code and then check at startup that the module definition of the same space was no larger.  Either way, there should be a comment in the module code telling developers to make sure this runtime type is big enough, if they change the module definition of the same space.